packages/antd/src/not-antd/LocationDependentArea.tsx

```diff
@@ -109,7 +109,11 @@ export class LocationDependentArea<DataType> extends React.Component<LocationDep
         this.debugLog('wantedChild is', wantedChild);
         if (mountAll) {
             this.debugLog('Rendering all children at once');
-            return <div>{this._states._subStatesInContext.map(s => this.renderSubState(s, s !== wantedChild))}</div>;
+            return this._states._subStatesInContext.map((s, i) => (
+                <div key={`${i}`} style={{ display: s !== wantedChild ? 'none' : undefined }}>
+                    {this.renderSubState(s, s !== wantedChild)}
+                </div>
+            ));
```

Comment:
Does it work to return an array of HTML elements from render? Older Reach versions demanded a single element. (Hence the `div` element wrapped around the array of individual `div`s.")

Reply:
With the current code version, we are doing two things in parallel for hiding the unwanted children:
 - Adding "display:none" here
 - Also passing in the `invisible` prop to the sub-components.

These are redundant, but let's leave it like this now, because it doesn't hurt.

Later I will create two separate options for the two separate behaviors.